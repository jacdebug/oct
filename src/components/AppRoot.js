import React from 'react';
import { ThemeProvider } from 'styled-components';
import MarketsContainer from '../containers/MarketsContainer';
import Header from './Header';
import Footer from './Footer';
import theme from '../constants/colors';

const AppRoot = ({ ...props }) =>
 <ThemeProvider theme={theme}>
  <div className="App">
   
    <Header {...props} />
    <MarketsContainer {...props} />
    <Footer {...props} />
    
  </div>
  </ThemeProvider>;

export default AppRoot;
