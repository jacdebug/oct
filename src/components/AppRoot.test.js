import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AppRoot from './AppRoot';

it('renders without crashing', () => {
  shallow(<AppRoot />);
});

it('renders a snapshot', () => {
  const tree = shallow(<AppRoot />);
  expect(toJson(tree)).toMatchSnapshot();
});
