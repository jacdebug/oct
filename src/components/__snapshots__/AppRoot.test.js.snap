// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<ThemeProvider
  theme={
    Object {
      "tcolor1": "#2b364b",
      "tcolor2": "#707886",
      "tcolor3": "#ffc952",
      "tcolor4": "#fff",
      "tcolor5": "#1c2637",
    }
  }
>
  <div
    className="App"
  >
    <Component />
    <Connect(MarketsContainer) />
    <Component />
  </div>
</ThemeProvider>
`;
